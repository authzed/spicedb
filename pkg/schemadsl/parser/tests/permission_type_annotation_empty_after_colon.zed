definition mydefinition {
	relation viewer: user
	permission view: = viewer
}