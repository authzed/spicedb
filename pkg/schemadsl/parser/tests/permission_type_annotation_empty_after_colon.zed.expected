NodeTypeFile
  end-rune = 76
  input-source = permission type annotation empty after colon test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = mydefinition
      end-rune = 76
      input-source = permission type annotation empty after colon test
      start-rune = 0
      child-node =>
        NodeTypeRelation
          end-rune = 47
          input-source = permission type annotation empty after colon test
          relation-name = viewer
          start-rune = 27
          allowed-types =>
            NodeTypeTypeReference
              end-rune = 47
              input-source = permission type annotation empty after colon test
              start-rune = 44
              type-ref-type =>
                NodeTypeSpecificTypeReference
                  end-rune = 47
                  input-source = permission type annotation empty after colon test
                  start-rune = 44
                  type-name = user
        NodeTypePermission
          end-rune = 74
          input-source = permission type annotation empty after colon test
          relation-name = view
          start-rune = 50
          compute-expression =>
            NodeTypeIdentifier
              end-rune = 74
              identifier-value = viewer
              input-source = permission type annotation empty after colon test
              start-rune = 69
          type-annotations =>
            NodeTypeTypeAnnotation
              end-rune = 65
              input-source = permission type annotation empty after colon test
              start-rune = 67
              child-node =>
                NodeTypeError
                  end-rune = 65
                  error-message = Expected identifier, found token TokenTypeEquals
                  error-source = =
                  input-source = permission type annotation empty after colon test
                  start-rune = 67
                NodeTypeError
                  end-rune = 65
                  error-message = Expected type identifier in type annotation
                  error-source = =
                  input-source = permission type annotation empty after colon test
                  start-rune = 67